apps/studio/src/features/editing-experience/components/RawJsonEditor.tsx

```diff
@@ -0,0 +1,121 @@
+import {
```

Comment:
Can help explain why is editing the title not possible?

Reply:
The title right now is being stored as a separate column in the Resources table, whereas the content is taken from the Blob table, so editing the title directly on the content right now does not propagate the change back to the Resources table.

If it's a more common request from the migrators then maybe we can add this functionality in the future.